### PR TITLE
Organize cmd package

### PR DESCRIPTION
**Pull request by lcarva**
The cmd package is now organized into sub-package (and sub-directories) that match the CLI structure.

This also moves the create-pr and replace commands under the experimental sub-command to better represent their state.

The `fs` and `withFs` helper functions have been moved to the generic internal/utils package to avoid cyclic imports. It also seems to be a better home for it as it may be used in other parts of the code base.

https://issues.redhat.com/browse/HACBS-1736

Structure looks like this:
```
cmd
├── experimental
│   ├── create_pr.go
│   ├── experimental.go
│   ├── replace.go
│   └── replace_test.go
├── fetch
│   ├── fetch_commit.go
│   ├── fetch.go
│   ├── fetch_k8s_resource.go
│   └── fetch_policy.go
├── inspect
│   ├── inspect.go
│   └── inspect_policy.go
├── root.go
├── track
│   ├── track_bundle.go
│   ├── track_bundle_test.go
│   └── track.go
├── validate
│   ├── image.go
│   ├── image_test.go
│   ├── pipeline.go
│   ├── pipeline_test.go
│   ├── test_application_snapshot.json
│   └── validate.go
└── version
    ├── version.go
    └── version_test.go
```

### CODE REVIEW

File: cmd/experimental/experimental.go [hunk 1]
```diff
@@ -14,22 +14,20 @@
 //
```
Comment (lcarva) [inline cmd/experimental/experimental.go:14]: GitHub is drunk. `cmd/experimental/experimental.go` is a new file, not a move from `cmd/io.go`. That one has been deleted.

**Review by simonbaird — APPROVED** [review]
Lgtm I guess. Better make the tests pass first!

**Comment by lcarva** [comment]
There was an issue that the acceptance tests caught :tada: 

It now seems to be hitting the random 404 errors. The tests do work on my machine :tm:

**Comment by zregvart** [comment]
> It now seems to be hitting the random 404 errors. The tests do work on my machine tm

I retry for those, this is the ever looming [HACBS-1354](https://issues.redhat.com/browse/HACBS-1354).

**Review by zregvart — APPROVED** [review]
Thanks! We need to get this merged to reduce the number of conflicts.

**Pull request merged**